web/src/components/CreateTestPlugins/Rest/steps/RequestDetails/RequestDetailsUrlInput.tsx

```diff
@@ -35,8 +36,23 @@ const RequestDetailsUrlInput = ({showMethodSelector = true, shouldValidateUrl =
         <Form.Item
           name="url"
           rules={[
-            {required: true, message: 'Please enter a request url'},
-            shouldValidateUrl ? {type: 'url', message: 'Request url is not valid'} : {},
+            {
+              type: 'url',
```

Comment:
Should we move this to the `web/src/utils/Validator.ts` file instead?

Reply:
i have swapped isValidUrl to the Validator.url